guide/onboarding/introduction.md

```diff
@@ -1,85 +0,0 @@
----
-layout: guide
-title: Onboarding
-description: An overview of some of the concepts to consider when building onboarding experiences.
-has_children: true
-nav_order: 4
-permalink: /guide/onboarding/introduction/
-main_classes: -no-top-padding
-image: https://bitcoin.design/assets/images/guide/onboarding/onboarding-preview.png
----
-
-<!--
-
-Editor's notes
-
-A brief introduction and summary of all pages in this section. The idea is that readers
-scan this page to get an overview of the section and then decide which topics to dive into.
-
-Illustration sources:
-
-- https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=0%3A1
-
--->
-
-{% include picture.html
-   image = "/assets/images/guide/onboarding/onboarding-main.png"
-   retina = "/assets/images/guide/onboarding/onboarding-main@2x.png"
-   mobile = "/assets/images/guide/onboarding/onboarding-main-mobile.png"
-   mobileRetina = "/assets/images/guide/onboarding/onboarding-main-mobile@2x.png"
-   alt-text = "Onboarding landing page header"
-   width = 1600
-   height = 600
-   layout = "full-width"
-%}
-
-# Onboarding
-
-Onboarding, the first use, can make or break someone’s experience of a product. Therefore it is important to identify your target audience and design specifically with its context, expectations, and needs in mind.
```

Comment:
Some good top-level info about onboarding here. Wonder if we could re-use this somewhere instead of deleting it.

Reply:
I made backups of all these pages fyi, also saved in Git history but just in case. Was keen to re-purpose a lot of that content.